editor/gui/editor_bottom_panel.cpp

```diff
@@ -200,6 +201,16 @@ void EditorBottomPanel::remove_item(Control *p_item) {
 			break;
 		}
 	}
+
+	if (was_visible) {
+		// Open the first panel to ensure that if the removed dock was visible, the bottom
+		// panel will not collapse.
+		_switch_to_item(true, 0);
```

Comment:
I wonder if it's better to close the panel instead 🤔
I don't see why would you want to open Output in this case.

Reply:
I'm 50/50 on this one personally. I'd say we should keep this PR's behavior as-is and change it if there's significant pushback.

Reply:
I tend to agree with @Calinou, this behavior prevent the closing of the bottom panel when it's already opened. But it's really a personal taste at this point.